README.md

```diff
@@ -1,4 +1,4 @@
-ZClassic 2.0.3
+Zcash 2.0.4
```

Comment:
Why is it ZCash 2.0.4 and not ZClassic 2.0.4

Reply:
updated